lgc/test/StackBoundsCheck.lgc

```diff
@@ -0,0 +1,128 @@
+; RUN: lgc -mcpu=gfx900 -stop-after=lgc-bounds-check-memory - <%s | FileCheck %s
+
+; lgc code generated with amdllpc llpc/test/shaderdb/OpLoad_StackBoundsCheck_struct.pipe -emit-lgc
+
+; CHECK:      = fmul reassoc nnan nsz arcp contract afn float %[[#]], 2.550000e+02
```

Comment:
Consider interleaving these checks with or moving them closer to the input IR. Right now we have to scroll a lot to 'align' the input and output IR and see what's expected to change.